R/data.R

```diff
@@ -1,82 +1,103 @@
+#' MEMC MEND model configuration based on \href{https://doi.org/10.1890/12-0681.1}{Wang et al. 2013}
 #'
-#'
-#' @format an object created form \code{configure_model}.
+#' @format an object created by \code{configure_model}.
 #' \describe{
 #' \item{name}{MEND}
 #' \item{params}{the default MEMC parameter table}
 #' \item{state}{the default inital state values for MEMC}
 #' \item{carbon_pools_func}{carbon pools based on Wang et al. 2013}
-#' \item{carbon_fluxes_func}{carbon fluxes based on Wang et al. 2013}
-#' }
+#' \item{carbon_fluxes_func}{carbon fluxes based on Wang et al. 2013.}}
 #' @family MEMC configurations
+#' @name MEND_model
 #' @references \href{https://doi.org/10.1890/12-0681.1}{Wang et al. 2013}
 #' @examples
 #' MEND_model
 #' solve_model(mod = MEND_model, time = 1:10)
 "MEND_model"
 
-#' MEMC model configurations for MEND Wang et al. 2013
-#'
+#' MEMC COMISSION model configuration based on  \href{https://doi.org/10.1016/j.soilbio.2020.107912}{Ahrens et al. 2020}
 #'
-#' @format an object created form \code{configure_model}.
+#' @format an object created by \code{configure_model}.
 #' \describe{
 #' \item{name}{COMISSION}
 #' \item{params}{the default MEMC parameter table}
 #' \item{state}{the default inital state values for MEMC}
 #' \item{carbon_pools_func}{carbon pools based on Wang et al. 2013}
-#' \item{carbon_fluxes_func}{TBD}
+#' \item{carbon_fluxes_func}{carbon fluxes based on Wang et al. 2013. The COMISSION_model MEMC
+#' configuration uses RMM kinetics for DOMdecomp, MM for  POMdecomp, and LM for MBdecay.
+#' see \code{\link{kinetics}} for more details.}
 #' }
 #' @family MEMC configurations
+#' @name COMISSION_model
+#' @references \href{https://doi.org/10.1016/j.soilbio.2020.107912}{Ahrens et al. 2020}
 #' @references \href{https://doi.org/10.1890/12-0681.1}{Wang et al. 2013}
-#' @references \href{https://doi.org/10.1890/12-0681.1}{update with the comission documetation}
 #' @examples
 #' COMISSION_model
 #' solve_model(mod = COMISSION_model, time = 1:10)
 "COMISSION_model"
 
-
-
-
+#' MEMC CORPSE model configuration based on \href{https://doi.org/10.1038/nclimate2436}{Sulman et al. 2014}
+#'
+#' @format an object created by \code{configure_model}.
+#' \describe{
+#' \item{name}{CORPSE}
+#' \item{params}{the default MEMC parameter table}
+#' \item{state}{the default inital state values for MEMC}
+#' \item{carbon_pools_func}{carbon pools based on Wang et al. 2013}
+#' \item{carbon_fluxes_func}{carbon fluxes based on Wang et al. 2013. The CORPSE_model MEMC
+#' configuration uses RMM kinetics for DOMdecomp, LM for POMdecomp, and LM for MBdecay.
+#' see \code{\link{kinetics}} for more details.}
+#' }
+#' @family MEMC configurations
+#' @name CORPSE_model
+#' @references \href{https://doi.org/10.1038/nclimate2436}{Sulman et al. 2014}
+#' @references \href{https://doi.org/10.1890/12-0681.1}{Wang et al. 2013}
+#' @examples
+#' CORPSE_model
+#' solve_model(mod = CORPSE_model, time = 1:10)
+"CORPSE_model"
```

Comment:
We have corpse, commission, and mend, do we want to add any other model configurations?

Reply:
No additional configurations at this point. Will need to enable toy model---user-defined configuration